uw-frame-components/portal/notifications/partials/notification-bell.html

```diff
@@ -44,6 +44,9 @@
           {{button.label}}
         </md-button>
       </div>
+      <div class="dismiss-priority hidden-xs">
+        <md-button ng-if="priority.dismissable" ng-click="dismiss(priority, true)"><i class='fa fa-times' aria-label="Dismiss this priority notification"></i></md-button>
```

Comment:
The aria-label attribute should be part of md-button, not the <i> element. Not sure if screen readers will pick it up as is.